src/common/schedulers/slurm_commands.py

```diff
@@ -362,15 +371,21 @@ def _parse_nodes_info(slurm_node_info):
         "State": "state",
         "Partitions": "partitions",
         "Reason": "reason",
+        "SlurmdStartTime": "slurmdstarttime",
     }
 
-    node_info = slurm_node_info.split("######")
+    node_info = slurm_node_info.split("######\n")
     slurm_nodes = []
     for node in node_info:
-        lines = node.strip().splitlines()
```

Comment:
why did you remove the strip()? it could be useful in case for any reason there are trailing whitespaces in the Slurm output

Reply:
I just checked. Actually this has more to do with the output of our `SCONTROL_OUTPUT_AWK_PARSER` than anything coming from `scontrol show node`: this `strip()` was removing the trailing `\n` at the end each node line, which was there because we were splitting by `######` and not `######\n`.

Now, since we split according to `######\n`, it's not strictly necessary. Actually, the `strip()` in that position would remove trailing spaces **only** in the last field we parse in our awk parser, which currently is `Reason`. If in the future we start parsing additional fields (`Comment` or `Extra`, for example), the `strip()` will start affecting a field different than the reason.

Given this information, I would be more inclined towards not having the `strip()` at all.